diff -ruN ../at-3.1.8.orig/at.c ./at.c
--- ../at-3.1.8.orig/at.c	Mon Jan 14 17:59:08 2002
+++ ./at.c	Mon Jan 14 17:59:48 2002
@@ -292,7 +292,7 @@
 	 * bit.  Yes, this is a kluge.
 	 */
 	cmask = umask(S_IRUSR | S_IWUSR | S_IXUSR);
-	if ((fd = open(atfile, O_CREAT | O_TRUNC | O_WRONLY, S_IRUSR)) == -1)
+	if ((fd = open(atfile, O_CREAT | O_EXCL | O_TRUNC | O_WRONLY, S_IRUSR)) == -1)
 	    perr("Cannot create atjob file %.500s", atfile);
 
 	if ((fd2 = dup(fd)) < 0)
