diff -Nru at-3.1.8/at.1.in at-3.1.8.new/at.1.in
--- at-3.1.8/at.1.in	Sun Mar 28 19:48:00 1999
+++ at-3.1.8.new/at.1.in	Sun Mar 28 19:46:41 1999
@@ -248,7 +248,11 @@
 .br
 .I @ETCDIR@/at.deny
 .SH SEE ALSO
-cron(1), nice(1), sh(1), umask(2), atd(8)
+.BR cron (1),
+.BR nice (1),
+.BR sh (1),
+.BR umask (2),
+.BR atd (8).
 .SH BUGS
 The correct operation of
 .B batch
diff -Nru at-3.1.8/at.deny at-3.1.8.new/at.deny
--- at-3.1.8/at.deny	Sun Feb 23 21:24:34 1997
+++ at-3.1.8.new/at.deny	Sun Mar 28 19:46:41 1999
@@ -1,13 +1,24 @@
-nobody
+alias
+backup
 bin
 daemon
-sys
+ftp
+games
+gnats
+guest
+irc
 lp
-sync
 mail
-news
-uucp
-games
 man
-guest
-ftp
+nobody
+operator
+proxy
+qmaild
+qmaill
+qmailp
+qmailq
+qmailr
+qmails
+sync
+sys
+www-data
diff -Nru at-3.1.8/at_allow.5 at-3.1.8.new/at_allow.5
--- at-3.1.8/at_allow.5	Sun Mar 28 19:48:00 1999
+++ at-3.1.8.new/at_allow.5	Sun Mar 28 19:46:41 1999
@@ -29,4 +29,8 @@
 .I /etc/at/at.deny
 is checked.
 .SH "SEE ALSO"
-at(1), atrun(1), cron(1), crontab(1), atd(8)
+.BR at (1),
+.BR atrun (1),
+.BR cron (8),
+.BR crontab (1),
+.BR atd (8).
diff -Nru at-3.1.8/atd.8.in at-3.1.8.new/atd.8.in
--- at-3.1.8/atd.8.in	Sun Mar 28 19:48:00 1999
+++ at-3.1.8.new/atd.8.in	Sun Mar 28 19:46:41 1999
@@ -46,6 +46,11 @@
 is installed as
 .B /usr/sbin/atrun
 for backward compatibility.
+.SH WARNING
+.B atd
+won't work if its spool directory is mounted via NFS even if
+.I no_root_squash
+is set.
 .SH FILES
 .I @ATJBD@
 The directory for storing jobs; this should be mode 700, owner
@@ -61,9 +66,15 @@
 .B at
 system.
 .SH "SEE ALSO"
-at(1), atrun(1), cron(1), crontab(1), syslog(3), at_deny(5), at_allow(5)
+.BR at (1),
+.BR atrun (1),
+.BR cron (8),
+.BR crontab (1),
+.BR syslog (3),
+.BR at.deny (5),
+.BR at.allow(5).
 .SH BUGS
 The functionality of 
 .B atd
 should be merged into
-.BR cron(1) .
+.BR cron (8) .
--- at-3.1.8/batch	Thu Jan  1 01:00:00 1970
+++ at-3.1.8.new/batch	Sun Mar 28 19:46:41 1999
@@ -0,0 +1,4 @@
+#! /bin/sh
+prefix=/usr
+exec_prefix=${prefix}
+exec ${exec_prefix}/bin/at -qb now "$@"
